examples/embed-directive/embed-directive.go

```diff
@@ -0,0 +1,51 @@
+// Starting in Go version 1.16+, Go supports use of `//go:embed`
+// which is a directive to embed files and folders into
+// the application binary by the compiler.
+
+// Since Go does not have preprocessor or macros like C type of languages,
```

Comment:
I'll make this all somewhat briefer, providing only the necessary information for the example at hand, w/o the background: no need to mention Go version, or the preprocessor vs. pragmas, etc.

Reply:
I was trying to make it clear to people that is not familiar with go directive, and would mistakenly think the `//` comment before `go:embed` is not needed. Will make it simpler tho and do you think providing a link on this topic would be better?

Reply:
Sure, just make it very brief (and a link SGTM). Like, "//go:embed is a compiler directive that ..."

Reply:
Tweaked now.